src/passkey_child/passkey_child_common.c

```diff
@@ -520,6 +568,14 @@ authenticate(struct passkey_data *data)
     int count = 0;
     errno_t ret;
 
+    pk_data.type = 0;
```

Comment:
Hi,

wouldn't it be easier to just initialize `pk_data` completely with `struct pk_data_t pk_data = { 0 };`  in line 566?

bye,
Sumit

Reply:
Done. Thanks for the review!